.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,6 +3,7 @@
 - [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
 - [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
 - [ ] Have you successfully run `brew style` with your changes locally?
+- [ ] Have you successfully run `brew typecheck` with your changes locally?
 - [ ] Have you successfully run `brew tests` with your changes locally?
 - [ ] Have you successfully run `brew man` locally and committed any changes?
```

Comment:
```suggestion
- [ ] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew typecheck` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?
- [ ] Have you successfully run `brew man` locally and committed any changes?
```

- Would it be good to use the `brew typecheck` option that generates/changes files here?
- What about a single (Bash?) command that runs `brew style --fix`, `brew typecheck --update`, `brew man`, `brew tests` and commits the results if they all pass?

Reply:
`brew typecheck --update` currently only updates the files but does not actually type check. Given `brew typecheck --udpate` mostly only updates RBI files for gems, I don't think `--update` will fix any errors that occur as a result of editing Homebrew files.

Reply:
> What about a single (Bash?) command

I don't think I would personally run such a command. E.g. if you know only style is failing, `brew tests` will uselessly run for ages.

Reply:
Ok, fine with me for now 👍🏻